app/pandas.ipynb

```diff
@@ -547,13 +832,53 @@
     "collapsed": false
    }
   },
+  {
+   "cell_type": "markdown",
+   "source": [],
+   "metadata": {
+    "collapsed": false
+   }
+  },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
+   "execution_count": 4,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: total: 406 ms\n",
+      "Wall time: 471 ms\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": "4          51.8\n5          42.2\n6          42.0\n7          30.6\n8          34.4\n           ... \n9999995    46.6\n9999996    61.6\n9999997    64.8\n9999998    67.2\n9999999    74.2\nLength: 9999996, dtype: float64"
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# write your code here\n"
+    "def estimate_time():\n",
+    "    s_set = pd.Series(np.random.randint(0, 100, size=10000000))\n",
```

Comment:
with your range you receive 0...99 numbers, but in python code we have another range 1..100. Check additionally the difference between np.random.randint() and random.randint()